bin/messageformat.js

```diff
@@ -14,17 +14,14 @@ var fs = require('fs'),
     knownOpts = {
       help: Boolean,
       locale: [String, Array],
-      namespace: String
+      namespace: String,
+      'disable-plural-key-checks': Boolean
     },
     shortHands = {
       h: ['--help'],
       l: ['--locale'],
       n: ['--namespace']
     },
-    description = {
-      locale: "locale(s) to use [mandatory]",
-      namespace: "global object in the output containing the templates",
-    },
```

Comment:
This object was missed in the [refactor](https://github.com/messageformat/messageformat.js/commit/497b948894e4d9c6e55f4908de076d04864a20b9#diff-4e24a5453f5551ab3b7e0f852740ee7fL80)